koku/masu/database/presto_sql/reporting_ocpusagelineitem_daily_summary.sql

```diff
@@ -480,8 +483,18 @@ WHERE lids.source = {{source}}
     AND lids.node IS NOT NULL
     AND lids.data_source = 'Pod'
 GROUP BY lids.node, lids.usage_start, lids.source_uuid
+
+{% else %}
+
+SELECT 1
```

Comment:
I am completely open to a better solution for this, I was running into errors with trying to include a `;` inside of jinja's if block. If you do this:
```
{% if my_unleash_flag %}
<some sql here>
;
{% endif %}
```
You will the following error:
```
Jinja was looking for the following tags: 'elif' or 'else' or 'endif'.
```

However, since the unleash gate will only be in place until we notify our customers of the upcoming cost changes, I didn't want to sink too much time into the perfect solution for code that will be removed in the future. The `Select 1` allows me to move the `;` outside of the if statement making both sql & jinja happy. 

----

I also tried:
```
(Option One)
{% set my_string = ; %}
{{ my_string }}

(Option Two)
{{ ; | sqlsafe}}

(Option Three)
{{ autoescape false }}
;
{{ endautoescape }}
```

Reply:
I forgot to put context for why I included the else statement 🤦 

So, that was the first thing I tried, but it resulted in the following Trino error: 

```
TrinoUserError(type=USER_ERROR, name=SYNTAX_ERROR, message="line 12:1: mismatched input '<EOF>'.
```

I added the else statement so prevent the mismatched EOF caused moving the the `;` outside the if condition. So that both trino & jinja would be happy with the sql created by the template.